repository-ydb-v2/src/main/java/tech/ydb/yoj/repository/ydb/merge/ByEntityYqlQueriesMerger.java

```diff
@@ -121,9 +121,11 @@ public List<YdbRepository.Query<?>> getQueries() {
 
     private boolean needIgnoreQuery(EntityState entityState) {
         if (entityState.state == MergingState.UPSERT || entityState.state == MergingState.INSERT) {
-            Class<?> clazz = getEntityClass(entityState.query);
-            Entity.Id entityId = getEntityId(entityState.query);
-            RepositoryCache.Key key = new RepositoryCache.Key(clazz, entityId);
+            YqlStatement<?, ?, ?> srcStatement = convertQueryToYqlStatement(entityState.query);
+            Class<?> entityClass = srcStatement.getInSchemaType();
```

Comment:
minor thing: upsert and insert statements have the same input schema and output schema, so we *can* use `getInSchemaType()` here (and in fact, the `getEntityClass()` method that was previously called here, did the same thing...)